lib/lte_link_control/lte_lc_helpers.c

```diff
@@ -1040,12 +1056,19 @@ int parse_ncellmeas(const char *at_response, struct lte_lc_cells_info *cells)
 		}
 	}
 
+	if (incomplete) {
+		err = -E2BIG;
+		LOG_ERR("Buffer is too small; results incomplete: %d", err);
```

Comment:
This is copied from `parse_ncellmeas_cgi()`, but maybe a warning would be sufficient here.

Reply:
Yep, I'll make it `LOG_WRN` both here and in `parse_ncellmeas_cgi()`.